### PR TITLE
speedtest: Add 'noclear' flag to keep the generated data

### PR DESCRIPTION
**Pull request by vadmeste**
For mc support perf <alias>, the default behavior is to remove 
the generated objects when doing object speed test. Adding
 a flag to offer a possibility to keep the generated data, so we 
can have a way to quickly generate data of any size.

e.g.
mc support perf object <alias> --duration 30m --keep-data --size 1K

### CODE REVIEW

File: perf-object.go [hunk 1]
```diff
@@ -65,6 +65,7 @@ type SpeedtestOpts struct {
 	Autotune     bool          // Enable autotuning
 	StorageClass string        // Choose type of storage-class to be used while performing I/O
 	Bucket       string        // Choose a custom bucket name while performing I/O
+	KeepData     bool          // Avoid cleanup after running an object speed test
```
Comment (harshavardhana) [inline perf-object.go:68]:
```suggestion
	NoClear     bool          // Avoid cleanup after running an object speed test
```

File: perf-object.go [hunk 2]
```diff
@@ -97,6 +98,9 @@ func (adm *AdminClient) Speedtest(ctx context.Context, opts SpeedtestOpts) (chan
 	if opts.Autotune {
 		queryVals.Set("autotune", "true")
 	}
+	if opts.KeepData {
```
Comment (harshavardhana) [inline perf-object.go:101]:
```suggestion
	if opts.NoClear {
```

File: perf-object.go [hunk 3]
```diff
@@ -97,6 +98,9 @@ func (adm *AdminClient) Speedtest(ctx context.Context, opts SpeedtestOpts) (chan
...
+		queryVals.Set("keep-data", "true")
```
Comment (harshavardhana) [inline perf-object.go:102]:
```suggestion
		queryVals.Set("noclear", "true")
```

**Review by harshavardhana — CHANGES_REQUESTED** [review]

**Comment by vadmeste** [comment]
@harshavardhana it looks like this PR is overkill since a single 'warp' cli command is enough, what do you think?

**Comment by vadmeste** [comment]
We just this for this PR https://github.com/actions/go-versions/pull/70 to be merged to fix vulncheck complaints

**Comment by vadmeste** [comment]
@harshavardhana can you take a look ?

**Review by harshavardhana — APPROVED** [review]

**Pull request merged**